qml/DomainGraphLayout.qml

```diff
@@ -1265,18 +1303,48 @@ Item
 
     Timer {
```

Comment:
Timer should only start if the model is not empty.
Add time interval limit to avoid infinite growth?